styleframe/tests/style_frame_tests.py

```diff
@@ -674,3 +675,24 @@ def test_columns_setter(self):
         self.sf.columns = ['c', 'd']
         self.assertTrue(all(isinstance(col, Container) for col in self.sf.columns))
         self.assertEqual([col.value for col in self.sf.columns], ['c', 'd'])
+
+
+class ToExcel(unittest.TestCase):
+    """Tests related to `StyleFrame.to_excel()`.
+    """
+
+    def test_pandas_defaults(self):
+        """
+        """
+        # values that StyleFrame assume as pandas defaults
+        header, startcol, startrow, na_rep \
+            = StyleFrame._to_excel_pandas_defaults({})
+
+        # the "real" default values pandas use
+        sig = inspect.signature(pd.DataFrame.to_excel)
+
+        # compare them
+        self.assertEqual(header, sig.parameters['header'].default)
+        self.assertEqual(startcol, sig.parameters['startcol'].default)
+        self.assertEqual(startrow, sig.parameters['startrow'].default)
+        self.assertEqual(na_rep, sig.parameters['na_rep'].default)
```

Comment:
This test should be in the same class that tests the `StyleFrame` class.

```suggestion
    def test_to_excel_pandas_defaults(self):
        # values that StyleFrame assume as pandas defaults
        header, startcol, startrow, na_rep = StyleFrame._to_excel_pandas_defaults({})

        # the "real" default values pandas use
        sig = inspect.signature(pd.DataFrame.to_excel)

        # compare them
        self.assertEqual(header, sig.parameters['header'].default)
        self.assertEqual(startcol, sig.parameters['startcol'].default)
        self.assertEqual(startrow, sig.parameters['startrow'].default)
        self.assertEqual(na_rep, sig.parameters['na_rep'].default)

```